js/ojsxc.js

```diff
@@ -102,11 +102,8 @@
             jsxc.gui.roster.toggle();
         });
 
-        if ($('#contactsmenu').length > 0) {
-            $('#contactsmenu').before(div);
-        } else {
-            $('#settings').after(div);
-        }
+        $('form.searchbox').after(div);
```

Comment:
I am afraid that this is not specific enough. I would love to see something like `#header form.searchbox` to prevent potential misbehavior with other apps.